build/configs/rtl8730e/hello/defconfig

```diff
@@ -24,6 +24,7 @@ CONFIG_BUILD_FLAT=y
 # CONFIG_APP_BINARY_SEPARATION is not set
```

Comment:
Why are we renaming loadable and hello 8M configs only?
I see that there are 8M configs for others such as cxxtest, etc.. Better to have a standard convention.

Reply:
I wish to rename those two files at least to make them aligned with our private files.
The others are added here for specific testing purposes and may be renamed.
Would like to know the general opinion on the naming of the other files @sunghan-chang @kishore-sn @ewoodev